source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/td_ekinetic_lcao.cpp

```diff
@@ -185,6 +192,7 @@ void TDEkinetic<OperatorLCAO<TK, TR>>::cal_HR_IJR(const int& iat1,
 template <typename TK, typename TR>
 void TDEkinetic<OperatorLCAO<TK, TR>>::init_td(void)
 {
+    TD_Velocity::td_vel_op = &td_velocity;
     //calculate At in cartesian coorinates.
 	double l_norm[3]={GlobalC::ucell.a1.norm() ,GlobalC::ucell.a2.norm() ,GlobalC::ucell.a3.norm()};
```

Comment:
Can you avoid using GlobalC::ucell?

Reply:
Ok, I'll remove the other GlobalC or GlobalV in these files as well.